.isort.cfg

```diff
@@ -0,0 +1,16 @@
+[settings]
+profile=black
+force_grid_wrap=1
+multi_line_output=3
+honor_noqa=true
+float_to_top=true
+combine_as_imports=true
+force_sort_within_sections=true
+include_trailing_comma=true
+extra_standard_library=pytest
+known_first_party=web3,ens,ethpm
+known_third_party=lru,eth_tester
+line_length=88
+use_parentheses=true
+# skip `__init__.py` files because sometimes order of initialization is important
```

Comment:
This was a big change to the new ``isort``, it used to skip ``__init__.py`` files by default.

Reply:
the `skip` line below includes `__init__.py`, but  it looks like several `__init__.py` files got sorted, e.g. `web3/__init__.py`. Is that expected?

Reply:
Yeah I sorted them and these didn't conflict so I let them be. I just fixed the ones where the initialization order mattered.